src/include/simeng/OS/Process.hh

```diff
@@ -37,41 +44,60 @@ uint64_t alignToBoundary(uint64_t value, uint64_t boundary);
 
 /** The initial state of a SimOS Process, constructed from a binary executable.
  *
- * The constructed process follows a typical layout in memory:
+ * The constructed process follows a typical layout and has the following
+ * properties:
+ *
+ * a) Padding between each region is equal to the page size. (4096 bytes)
```

Comment:
SimEng will only work with 4096, the page size on system can vary

Reply:
we can alter page sizes in Constants.hh but that file isn't meant to be edited i.e you shouldn't be editing Constants.hh to configure your simulation. The reason I say that SimEng will only work with 4096 bytes is because I have only tested it with 4096 bytes. While the unification does allow us to specify different values, currently the page size value by default is 4096 and it should not be changed, until I have tested SimEng will other values. I want to be safe and ensure to no anomalies happen when we make page size dynamic. So in my opinion this should be kept as is.